src/deploy/table_friend_policy.sql

```diff
@@ -0,0 +1,28 @@
+BEGIN;
+
+GRANT INSERT, UPDATE, SELECT ON TABLE maevsi.friend TO maevsi_account;
```

Comment:
It's still up for discussion who'll be able to see a friends list of a user. I'll update this comment myself when the question is answered, it's just a marker for me to not forget.

Reply:
It appears that anonymous users should not be able to see a user's friend list.